docs/tutorial/monitoring-and-metrics.md

```diff
@@ -96,6 +96,65 @@ The server collects metrics for both the data-plane and the control-plane. These
 
 The name of the metrics and the kind of metrics that are exported will depend on the mode that the server is running in. For TCP tunnels the metric name starts with `tcp_` for HTTP tunnels this will be `http_`.
 
+You don’t need to be a Kubernetes user to take advantage of Prometheus. You can run it locally on your machine by [downloading the binary here](https://prometheus.io/download/).
```

Comment:
Can you also link to Grafana Cloud for hosted tunnels and to the blog post you wrote last week?

Reply:
I updated this.